admin/tabs/server/logViewer/LogDisplayModel.ts

```diff
@@ -45,6 +45,10 @@ export class LogDisplayModel extends HoistModel {
     @persist
     caseSensitive: boolean = false;
 
+    @bindable
+    @persist
```

Comment:
@persist not doing anything, but saving this thing as a preference when it should not be.